cmd/api/main.go

```diff
@@ -117,6 +119,31 @@ func main() {
 		log.Fatalf("Failed to connect to database: %v", err)
 	}
 
+	sqlDB, err := db.DB()
+
+	// Set DB connection limits
+	maxIdle := 10
+	maxOpen := 10
+	maxIdleString := os.Getenv("MAX_IDLE_CONNECTIONS")
+	maxOpenString := os.Getenv("MAX_OPEN_CONNECTIONS")
+	if maxIdleString != "" {
+		maxIdle, err = strconv.Atoi(maxIdleString)
+		if err != nil {
+			log.Fatalf("Failed to parse MAX_IDLE_CONNECTIONS: %v", err)
```

Comment:
I think this should be Errorf instead of fatal if we want to setup maxIdle in case of error.